tests/TranslatableTest.php

```diff
@@ -226,9 +226,9 @@ public function getNameAttribute($value) : string
     public function it_can_use_mutators_on_translated_attributes()
     {
         $testModel = new class() extends TestModel {
-            public function setNameAttribute($value) : string
+            public function setNameAttribute($value)
```

Comment:
I did not understand the goal of this test because the mutator was an accessor (it was only returning something). I change it to a real mutator, but let me know if anything is wrong in my comprehension.